src/SWP/Bundle/FixturesBundle/DataFixtures/PHPCR/LoadSeparateArticlesData.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Fixtures Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\FixturesBundle\DataFixtures\PHPCR;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use SWP\Bundle\ContentBundle\Document\Article;
+use SWP\Bundle\FixturesBundle\AbstractFixture;
+use SWP\Bundle\ContentBundle\Document\Route;
+
+class LoadSeparateArticlesData extends AbstractFixture implements FixtureInterface
```

Comment:
It's class name, and this one provides clean not connected to anything articles - so they can be used later for tests where we connect them to something (as we don't have api to create them) ;)